src/Commands/ProjectRepairCommand.php

```diff
@@ -158,7 +158,16 @@ public function repair(
             // Tell the CI service to start testing
             ->taskCIStartTesting()
                 ->provider($this->ci_provider)
-                ->environment($ci_env);
+                ->environment($ci_env)
+
+            ->addCode(
+                function ($state) use ($site_name) {
+                    $secretValues = [
+                        'token' => $this->git_provider->token($this->git_provider->tokenKey())
+                    ];
+                    $this->writeSecrets("{$site_name}.dev", $secretValues, false, 'buildtools-secrets.json');
```

Comment:
Make sure this is consistent with `build:project:create`

Reply:
Done